### PR TITLE
Quay Workflow Management

### PR DESCRIPTION
**Pull request by sabre1041**
GitHub Action to manage CoP Quay.io organization

May be dependent on [https://github.com/redhat-cop/infra-ansible/pull/465](https://github.com/redhat-cop/infra-ansible/pull/465)

### CODE REVIEW

File: ansible/requirements.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- src: https://github.com/redhat-cop/infra-ansible
+  version: master
```
Comment (oybed) [inline ansible/requirements.yml:2]: would probably want to lock this one on a release?

Comment (sabre1041) [reply ansible/requirements.yml:2]: Agreed. Need to merge the PR in the description first, then cut a release

Comment (sabre1041) [reply ansible/requirements.yml]: @oybed updated

**Pull request merged**